docs/dev/new_check_howto.md

```diff
@@ -300,9 +300,59 @@ The `logos` folder must contain **three** images with filenames and sizes that m
+| `doc_link`          | String          | Mandatory          | Link to the documentation page of the integration. Format is `https://docs.datadoghq.com/integrations/<INTEGRATION_NAME>`                                                                                                          |
+| `guid`              | String          | Mandatory          | Unique ID for the integration. [Generate a UUID][13]                                                                                                                                                                               |
+| `is_public`         | Boolean         | Mandatory          | If set to `false` the integration `README.md` content is not indexed by bots in the Datadog public documentation.                                                                                                                  |
+| `maintainer`        | String          | Mandatory          | Email of the owner of the integration.                                                                                                                                                                                             |
+| `manifest_version`  | String          | Mandatory          |                                                                                                                                                                                                                                    |
+| `name`              | String          | Mandatory          | Unique name for the integration. Use the folder name for this parameter.                                                                                                                                                           |
+| `public_title`      | String          | Mandatory          | Title of the integration displayed on the documentation. Should follow the following format: `Datadog-<INTEGRATION_NAME> Integration`                                                                                              |
+| `short_description` | String          | Mandatory          | This text -Maximum 80 characters- appears at the top of the integration tile as well as the integration's rollover text on the integrations page.                                                                                  |
+| `support`           | String          | Mandatory          |                                                                                                                                                                                                                                    |
+| `supported_os`      | Array of String | Mandatory          | List of supported OSs. Choose between `linux`,`mac_os`, and `windows`.                                                                                                                                                             |
```

Comment:
`Choose among` maybe since there can be all of them ? Also, missing space after `,` in `linux`,`mac_os`